pkg/virt-controller/services/vm.go

```diff
@@ -192,13 +192,13 @@ func (v *vmService) StartMigration(migration *corev1.Migration, vm *corev1.VM, s
 	if err != nil {
 		return err
 	}
-	_, err = v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).Create(job)
+	_, err = v.KubeCli.CoreV1().Pods(migration.GetObjectMeta().GetNamespace()).Create(job)
 	return err
 }
 
 func (v *vmService) GetMigrationJob(migration *corev1.Migration) (*v1.Pod, bool, error) {
 	selector := migrationJobSelector(migration)
-	podList, err := v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).List(selector)
+	podList, err := v.KubeCli.CoreV1().Pods(migration.ObjectMeta.Namespace).List(selector)
```

Comment:
Yes. I'm not sure about that either.  In this first patch, I didn't see the use of GetObjectMeta(), so I just used ObjectMeta. Later, I used GetObjectMeta() if there was no previous use of ObjectMeta.. but honestly I'm not sure. If it does the same thing, I'll just change it to either one of these.